src/main/scala/zio/s3/UploadOptions.scala

```diff
@@ -0,0 +1,31 @@
+package zio.s3
+
+import software.amazon.awssdk.services.s3.model.{ ObjectCannedACL, PutObjectRequest }
+
+import scala.jdk.CollectionConverters._
+
+/**
+ * The options of the multipart upload and the put object request.
+ * @param metadata the user-defined metadata without the "x-amz-meta-" prefix
+ * @param cannedAcl a canned acl, defaults to "private"
+ * @param contentType the content type of the object (application/json, application/zip, text/plain, ...)
+ */
+case class UploadOptions(
```

Comment:
can you move this into settings.scala

Reply:
https://github.com/zio/zio-s3/pull/117/commits/bda1d842c0e9c8f75efb4e3e1084fc3270b79a96